Cargo.toml

```diff
@@ -70,4 +70,5 @@ uuid = { version = "1" }
 async-trait = { version = "0.1" }
 futures = { version = "0.3" }
 tokio = { version = "1" }
+async-channel = { version = "2.2" }
```

Comment:
As far as I can tell this is only used in one narrow slice within the core crate (datafusion-enabled writes) so I don't believe we need to sprout this dependency at the top-level